src/main/resources/db/migration/V7__Adding_New_Data.sql

```diff
@@ -0,0 +1,498 @@
+
+INSERT INTO Job_Roles (roleName, description, responsibilities, linkToJobSpec, capability, band, closingDate, status, positionsAvailable) 
```

Comment:
1. There still remains a scenario where apostrophes  can be inserted via RoleBulkImport and therefore you need to work with Jemima  to ensure she is correctly encoding the data into the database and put validation in the front and the back-end to prevent special characters entering the database via the csv route.

Reply:
Fixed